src/main/java/com/nylas/TimeSlot.java

```diff
@@ -9,6 +10,7 @@ public class TimeSlot {
 	// The availability endpoint uses these terms instead
 	private Long start;
 	private Long end;
+	private List<String> emails;
```

Comment:
You did not initialize this list, will it return null or crash the service when you call toString function? 
Usually people initialize it to empty list when they initialize a class.